pagerduty/resource_pagerduty_extension.go

```diff
@@ -49,6 +47,7 @@ func resourcePagerDutyExtension() *schema.Resource {
 			},
 			"config": {
 				Type:             schema.TypeString,
+				Computed:         true,
```

Comment:
Change was also covered in this PR, #95

Reply:
Sure, it comes from the same commit: https://github.com/terraform-providers/terraform-provider-pagerduty/pull/96/commits

Once #95 is merged, this change will no longer appear here.